site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -6,38 +6,38 @@ coauthor = ""
 forum = 0
 +++
 
-Greetings yuz-ers. While this monthly report will not be as extensive as others, we have several critical changes to discuss. Roll the tape!
+Greetings yuz-ers. This time around, we're covering small and incremental improvements to yuzu. Rest assured, we also have some major rewrites and improvements in the works, and we'll touch on those near the end. Roll the tape!
 
 <!--more--> 
 
 ## End of support for EOL Windows versions
 
 Let’s first address the elephant in the room, shall we?
 
-While working on dynarmic and kernel emulation, including improving the compatibility of 4 thread CPU systems, we made changes to dynarmic and fastmem that broke support for Windows 10 revision 1803 and older, including Windows 7 and Windows 8/8.1.
+While working on dynarmic and kernel emulation, including improving the compatibility of 4 thread CPU systems, we made changes to [dynarmic](https://github.com/merryhime/dynarmic) and [fastmem](https://yuzu-emu.org/entry/yuzu-fastmem/) that broke support for Windows 10 revision 1803 and older, including Windows 7 and Windows 8/8.1.
 
 While it was a purely accidental change, this was warned with the introduction of fastmem.
```

Comment:
Maybe use "notified" or "announced" or "informed" or "made clear" instead of warned?